src/app.js

```diff
@@ -180,23 +180,16 @@ export function app(app) {
     }
   }
 
-  function updateElementData(element, oldData, data, cb) {
+  function updateElementData(element, oldData, data) {
     for (var name in merge(oldData, data)) {
       var value = data[name]
       var oldValue = oldData[name]
 
-      if (
-        value !== oldValue &&
-        value !== element[name] &&
```

Comment:
@zaceno Yep... because it's kind of difficult to test. My bad. We need this because there are times when the DOM is already synced with `value`, so we don't care if `value !== oldValue`. 

For example, an input text, a check box, etc.